jobs/JREGIONAL_NEXUS_GFS_SFC

```diff
@@ -103,7 +109,8 @@ DATA="${DATA:-${COMIN}${SLASH_ENSMEM_SUBDIR}/tmp_NEXUS_GFS_SFC}"
 #-----------------------------------------------------------------------
 #
 if [ "${RUN_ENVIR}" = "nco" ]; then
-  export GFS_SFC_STAGING_DIR="${GFS_SFC_STAGING_DIR:-${DATA}}"
+  # export GFS_SFC_STAGING_DIR="${DATA}"
```

Comment:
Remove this line if not necessary.

Reply:
We will do a global development code clean up in a later commit. Because the AQM package in its current state, still under development.